modules/snooker/src/test/ts/module/ephox/snooker/test/TestGenerator.ts

```diff
@@ -18,10 +19,16 @@ export default (): SimpleGenerators => {
     return r;
   };
 
-  const replace = (name: string) => {
-    const r = 'h(' + name + ')_' + replaceCounter;
-    replaceCounter++;
-    return r;
+  const replace = (elem: SugarElement) => {
```

Comment:
```suggestion
  const replace = (elem: SugarElement<Node> | string) => {
```
This allows an element or string, so we need to reflect that here. We should also add the actual element type, otherwise it'll treat it as `any` which is something we want to eventually remove.